draft-ietf-ppm-dap.md

```diff
@@ -165,8 +165,17 @@ aggregator.
 
 16:
 
-- Bump draft-irtf-cfrg-vdaf-13 to 14 {{!VDAF}} and adopt changes to the
-  ping-pong API (\#705, \#718).
+- Bump draft-irtf-cfrg-vdaf-13 to 15 {{!VDAF}} and adopt changes to the
+  ping-pong API. (\*) (#705, #718)
```

Comment:
"#" gets interpreted by markdown ... I believe you want to escape it. Here and below.
```suggestion
  ping-pong API. (\*) (\#705, \#718)
```

Reply:
It doesn't appear to confuse GitHub's markdown renderer:

<img width="1046" height="303" alt="Image" src="https://github.com/user-attachments/assets/17ab5f9b-1853-4458-b841-27f635c6612d" />

I think the escapes are unnecessary.